
Waiting for JIT...
Traceback (most recent call last):
  File "/home/pedro/PINN/jaxpi/examples/geomI/main.py", line 39, in <module>
    app.run(main)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomI/main.py", line 31, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/home/pedro/PINN/jaxpi/examples/geomI/train.py", line 117, in train_and_evaluate
    log_dict = evaluator(state, batch, coords_fem, u_ref, v_ref)
                                       ^^^^^^^^^^
NameError: name 'coords_fem' is not defined